app/src/main/res/xml/preferences.xml

```diff
@@ -74,6 +74,17 @@
             android:title="@string/themes_title_settings"
             app:useSimpleSummaryProvider="true"/>
     </PreferenceCategory>
+    <PreferenceCategory
+        android:title="@string/need_help">
+        <Preference
+            android:key="docs"
+            android:icon="@drawable/ic_question"
+            android:title="@string/documentation"
+            android:summary="https://companion.home-assistant.io">
+            <intent android:action="android.intent.action.VIEW"
+                android:data="https://companion.home-assistant.io" />
+        </Preference>
```

Comment:
You can add `app:enableCopying="true"` here rather than setting it in the fragment.